newsfragments/30083.significant.rst

```diff
@@ -0,0 +1,13 @@
+Non-fixed cron timetables no longer account for changing in timezone.
```

Comment:
Is there a more descriptive term than "non-fixed"?  That doesn't have meaning for me, but maybe I'm alone on that.  At same time, I am not sure how to describe what it is.  Maybe "non-fixed-interval" or "not-constant-interval" or, instead of the double negative, "variable-interval"

Reply:
```suggestion
Variable-interval cron timetables no longer account for DST transition
```

also, is "account for DST transition" more accurate than "account for changing in timezone"?  when you say "changing in timezone" it sounds like you mean "changing the timezone" -- like, changing the timezone of the dag?